ext/ffi/lib.rs

```diff
@@ -996,19 +996,24 @@ fn ffi_parse_buffer_arg(
   // 5. Null: Very uncommon / can be represented by a 0.
 
   let pointer = if let Ok(value) = v8::Local::<v8::ArrayBuffer>::try_from(arg) {
-    let backing_store = value.get_backing_store();
-    &backing_store[..] as *const _ as *const u8
+    value.data()
   } else if let Ok(value) = v8::Local::<v8::ArrayBufferView>::try_from(arg) {
     let byte_offset = value.byte_offset();
-    let backing_store = value
+    let pointer = value
       .buffer(scope)
       .ok_or_else(|| {
         type_error("Invalid FFI ArrayBufferView, expected data in the buffer")
       })?
-      .get_backing_store();
-    &backing_store[byte_offset..] as *const _ as *const u8
+      .data();
+    if !pointer.is_null() && byte_offset > 0 {
+      // SAFETY: Pointer is non-null, and V8 guarantees that the byte_offset
+      // is within the buffer backing store.
+      unsafe { pointer.add(byte_offset) }
+    } else {
+      pointer
+    }
```

Comment:
```suggestion
    if !pointer.is_null() {
      // SAFETY: Pointer is non-null, and V8 guarantees that the byte_offset
      // is within the buffer backing store.
      unsafe { pointer.add(byte_offset) }
    } else {
      pointer
    }
```
Adding zero has no effect